### PR TITLE
Added sorting labels of tagged metrics

### PR DESCRIPTION
**Pull request by zhelyabuzhsky**
WARNING: You should remove all metrics in database first

```
./build/cli --config local/cli.yml -remove-all-metrics
```

### CODE REVIEW

File: filter/metrics_parser.go [hunk 1]
```diff
@@ -71,6 +72,23 @@ func ParseMetric(input []byte) (*ParsedMetric, error) {
...
+	keys := make([]string, 0, len(labels))
+	for key := range labels {
+		keys = append(keys, key)
+	}
+	sort.Strings(keys)
+
+	result += name
+
+	for _, key := range keys {
+		result += fmt.Sprintf(";%s=%s", key, labels[key])
```
Comment (kissken) [inline filter/metrics_parser.go:86]: Maybe better use strings.Builder for concatenate strings?

**Review by kissken — COMMENTED** [review]
Like and left one small comment

File: filter/metrics_parser.go (hunk 1 above)
Comment (zhelyabuzhsky) [reply filter/metrics_parser.go]: moved to strings.Builder

**Review by kissken — APPROVED** [review]
👍

**Review by kissken — APPROVED** [review]
👍

**Pull request merged**